src/sumo_store_riak.erl

```diff
@@ -489,8 +489,8 @@ new_doc(Doc, #state{conn = Conn, bucket = Bucket, put_opts = Opts}) ->
     undefined ->
       case update_map(Conn, Bucket, undefined, doc_to_rmap(Doc), Opts) of
```

Comment:
According to **Dialyzer**:
> The call sumo_store_riak:update_map(Conn::pid(), Bucket::{binary(),binary()}, 'undefined', {'map',[{{binary(),'counter' | 'flag' | 'map' | 'register' | 'set'},_}],[{{binary(),'counter' | 'flag' | 'map' | 'register' | 'set'},_}],[{binary(),'counter' | 'flag' | 'map' | 'register' | 'set'}],'undefined' | binary()}, Opts::['if_none_match' | 'if_not_modified' | 'return_body' | 'return_head' | {'dw','all' | 'default' | 'one' | 'quorum' | non_neg_integer()} | {'n_val',pos_integer()} | {'pw','all' | 'default' | 'one' | 'quorum' | non_neg_integer()} | {'sloppy_quorum',boolean()} | {'w','all' | 'default' | 'one' | 'quorum' | non_neg_integer()}]) will never return since it differs in the 3rd argument from the success typing arguments: (pid(), {binary(),binary()}, binary(), {'map',[{{_,_},_}],[{{_,_},_}],[{binary(),'counter' | 'flag' | 'map' | 'register' | 'set'}],'undefined' | binary()}, [atom() | tuple()])